react/graphql/AddProductTranslations.graphql

```diff
@@ -1,5 +1,6 @@
 mutation AddProductTranslations($translations: [MessagesFromProvider]!, $language: String!) {
   addProductTranslations(translations: $translations, language: $language) {
+    # Returns provider of failed translations
```

Comment:
Let's talk about this structure. We should not be using `data` to communicate failures